contracts/cairo/.env.example

```diff
@@ -0,0 +1,5 @@
+STARKNET_ACCOUNT=~/.starkli-wallets/account.json
+STARKNET_KEYSTORE=~/.starkli-wallets/keystore.json
+SN_RPC_URL=https://starknet-goerli.g.alchemy.com/v2/Oo-H7YBqtJrt4UCUYIad3KFYIPaDWXtB
```

Comment:
Please remove this we are exposing an API key

Reply:
Haha not actually, that is an invalid RPC URL, so the api key is invalid.
This was included as to show the format expected

Reply:
I would live it as `<starknet_rpc_url>' and include the format expected only for wallets with a comment next to it